walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -13,11 +13,92 @@
 // limitations under the License.
 
 package com.google.sps;
-
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 
 public final class FindMeetingQuery {
+
+  public static final int MINUTES_IN_DAY = 1440;
+
+  /**
+  * Returns possible meeting times on a certain day given a meeting request.
+  * Handles requests with only mandatory attendees, only optional attendees, both, and neither.
```

Comment:
what happens when there are no mandatory attendees? what happens when there are no events? please fill in these edge cases